# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/i915-dg1-enable.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.6/drivers/gpu/drm/i915/i915_pci.c.vanilla	2023-11-08 16:43:45.010027743 +0100
+++ linux-6.6/drivers/gpu/drm/i915/i915_pci.c	2023-11-08 16:44:56.935031287 +0100
@@ -672,7 +672,6 @@
 	DGFX_FEATURES,
 	.__runtime.graphics.ip.rel = 10,
 	PLATFORM(INTEL_DG1),
-	.require_force_probe = 1,
 	.platform_engine_mask =
 		BIT(RCS0) | BIT(BCS0) | BIT(VECS0) |
 		BIT(VCS0) | BIT(VCS2),
